packages/datatrak-web/src/api/queries/useSurveyUsers.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+
+import { useQuery } from 'react-query';
+import { Country, DatatrakWebSurveyUsersRequest } from '@tupaia/types';
+import { get } from '../api';
+import { Survey } from '../../types';
+
+export const useSurveyUsers = (
+  surveyCode?: Survey['code'],
+  countryCode?: Country['code'],
+  searchTerm?: string,
+) => {
+  return useQuery(
+    ['surveyUsers', surveyCode, countryCode, searchTerm],
+    (): Promise<DatatrakWebSurveyUsersRequest.ResBody> =>
+      get(`users/${surveyCode}/${countryCode}`, {
+        params: {
+          filter: searchTerm
+            ? {
+                full_name: {
+                  comparator: 'ilike',
+                  comparisonValue: `${searchTerm}%`,
```

Comment:
this feels like a backend concern to me since it's formatting sql filters. Could we just pass the search term in the request and do this is in the datatrak-web-server endpoint. That would make it more consistent with the Get Tasks Route as well